src/main/java/meteordevelopment/meteorclient/systems/modules/player/ChestSwap.java

```diff
@@ -120,6 +128,13 @@ private void equipElytra() {
 
     private void equip(int slot) {
         InvUtils.move().from(slot).toArmor(2);
+        if (closeInventory.get()) {
+            // Notchian clients send a Close Window packet with Window ID 0 to close their inventory even though there is never an Open Screen packet for the inventory.
+            var n = mc.getNetworkHandler();
+            if (n != null) {
+                n.sendPacket(new CloseHandledScreenC2SPacket(0));
```

Comment:
Why are you checking if the network handler is null, it never will be null here

Reply:
fried chicken